desktop/apps/editorial_features/routes.coffee

```diff
@@ -55,3 +55,9 @@ Articles = require '../../collections/articles.coffee'
   target = 'https://' + VANITY_BUCKET + '.s3.amazonaws.com' + '/' + req.params[0]
   proxy.web req, res, target: target, (err) ->
     res.redirect 301, '/articles' if err
+
+setVideoIndex = (curation, slug) ->
+  for section, i in curation.get 'sections'
+    if section.slug is slug
+      return i
+  return 0
```

Comment:
I know my temp code doesn't do this, but I think it makes more sense to do a 301 redirect here instead of defaulting to zero. We probably want people to share `artsy.net/venice-biennale` instead of `artsy.net/venice-biennale/some-fake-slug-that-accidentally-got-shared` 

Right after the success, you can `return res.redirect 301, '/venice-biennale'` if the slug isn't real